.dockerignore

```diff
@@ -1,7 +1,7 @@
 mnt/downloads/
 mnt/logs/
 passfile.txt
```

Comment:
Eventually, we should use `layeredconfig` to keep this out of the source tree and be able to source it from `~/.icgc-get/config.yaml`